test/cluster-store.js

```diff
@@ -19,7 +19,6 @@ if (cluster.isWorker) {
 
 describe('clustered socket.io', function() {
   before(setupWorkers);
-  after(stopWorkers);
```

Comment:
I am wondering why this is no longer needed. Was it a workaround for a bug that has been fixed meanwhile?